requirements.txt

```diff
@@ -3,7 +3,6 @@ semantic_version>=2.6.0
 argparse>=1.4.0
```

Comment:
I assume, all requirements should ensure a (likely) compatible version, e.g.
```
argparse>=1.4.0,<2
```
Otherwise, there may be breaking changes. (Well, there may still be breaking changes between minor version updates because people are not following SemVer closely. But at least, it's less likely.)

BTW: I just did `pip install -r requirements-dev.txt` and it installed `pytest 3.4.0`, while `pytest>=2.8.7` is in `requirements.txt`.

This is a general thing. Not necessary fro the PR to merge.

Reply:
In Python 3, you can run installed modules as scripts:
```
python3 -m pytest somedir/sometest.py.
``` 
It will uses the module installed in the current env AFAIK, i.e. the tool that's supposed to be installed via `requirements-dev.txt`.

Reply:
thanks! that works with `pytest`